### ISSUE TITLE
discord sends all messages to slack

### ISSUE THREAD

**Issue by tjeufoolen** [aubguillemette/discord2slack#4]
Currently the bot sends all discord messages to a specific slack channel, it doesn't check in what discord channel a message is send.

A quick fix for this is to check if the channel where the message is send in does equal the channel specified in the configuration section under 'DISCORD_CHANNELID'

Example:
```
if (message.channel.id === DISCORD_CHANNELID) {
        ...
}
```

**Comment by simisbeat** [comment]
I just hide bot from other channels, and add only in 1 where it can see messages

**Comment by aubguillemette** [comment]
Fixed in commit 9be105c, thanks
